bitsail-connectors/connector-kudu/src/main/java/com/bytedance/bitsail/connector/kudu/core/KuduFactory.java

```diff
@@ -127,6 +127,7 @@ public KuduSession getSession() {
 
       if (sessionConfig.getMutationBufferSize() != null) {
         kuduSession.setMutationBufferSpace(sessionConfig.getMutationBufferSize());
+        kuduSession.setErrorCollectorSpace(sessionConfig.getMutationBufferSize());
```

Comment:
It's not better to use different config for kudu, because the two config is very strong correlation.
Can seen the code : https://github.com/apache/kudu/blob/master/java/kudu-client/src/main/java/org/apache/kudu/client/AsyncKuduSession.java
When the flush mode set  `AUTO_FLUSH_BACKGROUND` , the flush will in the background thread,  if the flush row happen error, the error row will be added in the ErrorCollector,  So if we do not want to lose any error row , the ErrorCollectorSpace muse bigger than the MutationBufferSpace. The two config is equal default in the KuduSessuon.